src/main/java/seedu/address/logic/parser/DeleteCommandParser.java

```diff
@@ -19,8 +21,15 @@
      */
     public DeleteCommand parse(String args) throws ParseException {
         try {
-            Index index = ParserUtil.parseIndex(args);
-            return new DeleteCommand(index);
+            ArrayList<Index> indexArrayList = new ArrayList<Index>();
+            String[] indexArray = args.split(" ");
+            System.out.println("args: " + args);
+            System.out.println("indexArray.length: " + indexArray.length);
+            for (int i = 1; i < indexArray.length; i++) {
```

Comment:
Can we leave a comment why we start the loop from 1 instead of 0, as is convention?